test_vectors/pycryptodome_test_vectors/Hash/SHA3/ShortMsgSamples_cSHAKE128.txt

```diff
@@ -0,0 +1,19 @@
+# Testvectors converted from https://csrc.nist.gov/CSRC/media/Projects/Cryptographic-Standards-and-Guidelines/documents/examples/cSHAKE_samples.pdf
+
+# Sample 1
+NLen = 0
+SLen = 120
+Len = 32
+N = 00
+S = 456D61696C205369676E6174757265
+Msg = 00010203
+MD = C1C36925B6409A04F1B504FCBCA9D82B4017277CB5ED2B2065FC1D3814D5AAF5
+
+# Sample 2
+NLen = 0
+SLen = 120
+Len = 1600
+N = 00
+S = 456D61696C205369676E6174757265
+Msg = 000102030405060708090A0B0C0D0E0F101112131415161718191A1B1C1D1E1F202122232425262728292A2B2C2D2E2F303132333435363738393A3B3C3D3E3F404142434445464748494A4B4C4D4E4F505152535455565758595A5B5C5D5E5F606162636465666768696A6B6C6D6E6F707172737475767778797A7B7C7D7E7F808182838485868788898A8B8C8D8E8F909192939495969798999A9B9C9D9E9FA0A1A2A3A4A5A6A7A8A9AAABACADAEAFB0B1B2B3B4B5B6B7B8B9BABBBCBDBEBFC0C1C2C3C4C5C6C7
+MD = C5221D50E4F822D96A2E8881A961420F294B7B24FE3D2094BAED2C6524CC166B
```

Comment:
some more vectors for these two variants would be nice. 
unfortunately I haven't come across any.
maybe we can dump some from our SV UVM testbench?

Reply:
I have a bunch of test vector samples in `hw/dv/sv/test_vectors/vectors/xof/` from NIST, you can probably use those?

Reply:
Are they by change the same ones from here?
https://csrc.nist.gov/CSRC/media/Projects/Cryptographic-Standards-and-Guidelines/documents/examples/cSHAKE_samples.pdf

These are also from NIST...

Reply:
yup, unfortunately there aren't too many golden CSHAKE vectors...